src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -176,6 +176,17 @@ default String embedFooterIcon() {
         return "https://github.com/pajlads/DinkPlugin/raw/master/icon.png";
     }
 
+    @ConfigItem(
+        keyName = "rsnDenyList",
+        name = "RSN Deny List",
+        description = "Prevent notifications from the following player names (One name per line)",
```

Comment:
Nit: can we rename this to an ignorelist? To me it's more logical that notifications from these accounts would be ignored rather than denied.

Reply:
If we choose to use ignore, maybe it would make sense to structure the setting name differently altogether? "Ignored RSNs"?
Just tossing out curveballs, I'd be fine with either option

Reply:
Updated in https://github.com/pajlads/DinkPlugin/pull/114/commits/922c8661550ad5c9b77de35b98e254214443dc4d